androidApp/src/main/java/com/example/tasky/android/agenda/components/DetailsAttendeeSection.kt

```diff
@@ -0,0 +1,224 @@
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.compose.ui.window.Dialog
+import com.example.tasky.android.R
+import com.example.tasky.android.theme.Black
+import com.example.tasky.android.theme.Light2
+import com.example.tasky.android.theme.LightBlue
+import com.example.tasky.android.theme.MyApplicationTheme
+import com.example.tasky.model.agenda.Attendee
+import kotlinx.collections.immutable.ImmutableList
+import kotlinx.collections.immutable.toImmutableList
+
+@Composable
+fun DetailsAttendeeSection(
+    attendees: ImmutableList<Attendee>,
+    curTab: DetailsAttendeeSectionTabOption,
+    isEdit: Boolean,
+    onTabSelect: (DetailsAttendeeSectionTabOption) -> Unit,
+    onAddClick: (String) -> Unit,
+    onDeleteIconClick: (String) -> Unit,
+    creatorId: String,
+    modifier: Modifier = Modifier,
+) {
+    var showDialog by rememberSaveable {
+        mutableStateOf(false)
+    }
+
+    Box(modifier = modifier) {
+        Column {
+            Row(verticalAlignment = Alignment.CenterVertically) {
+                Text(stringResource(R.string.visitors), style = typography.headlineMedium, lineHeight = 16.sp, color = Black)
+                if (isEdit) {
+                    Spacer(modifier = Modifier.width(16.dp))
+                    IconButton(
+                        onClick = { showDialog = true },
+                        modifier =
+                            Modifier
+                                .background(Light2)
+                                .size(35.dp),
+                    ) {
+                        Icon(Icons.Outlined.Add, contentDescription = null, tint = LightBlue)
+                    }
+                }
+            }
+            Spacer(modifier = Modifier.height(24.dp))
+            DetailsAttendeeSectionTab(curTab = curTab, onTabSelect = onTabSelect)
+
+            if (curTab != DetailsAttendeeSectionTabOption.NOT_GOING) {
+                Spacer(modifier = Modifier.height(16.dp))
+                DetailsAttendeeSectionAttendeeList(
+                    titleId = R.string.going,
+                    attendees =
+                        attendees
+                            .filter {
+                                it.isGoing
+                            }.toImmutableList(),
+                    onDeleteIconClick = onDeleteIconClick,
+                    isEdit = isEdit,
+                    creatorId = creatorId,
+                )
+            }
+
+            if (curTab != DetailsAttendeeSectionTabOption.GOING) {
```

Comment:
This logic can be a bit simpler to follow by not using negate operators here, what do you think about the following, do you think it reads better?

```
if(curTab == ALL || curTab == GOING) {
  // render your going list
}

if(curTab == ALL || curTab == NOT_GOING) {
  // render your not going list
}
```

Personally, I find this easier to read and understand what the UI will render 😊